twake/frontend/src/app/features/users/api/online-user-realtime-api-client.ts

```diff
@@ -35,8 +37,20 @@ export const OnlineUserRealtimeAPI = (websocket: WebSocketService): OnlineUserRe
       .then(result => result || []);
   };
 
+  const setUserStatus = (id: GetUserRequestType): Promise<GetUserResponseType> => {
+    logger.debug(`Set user online status ${id}`);
+    return setUsersStatus([id]).then(response => response?.[0] || [id, false]);
+  };
+
+  const setUsersStatus = (ids: GetUsersRequestType = []): Promise<GetUsersResponseType> => {
+    logger.debug(`Set users statuses`);
+    return websocket
+      .get<GetUsersRequestType, GetUsersResponseType>('online:set', ids)
+      .then(result => result || []);
```

Comment:
You are not allowed to set the status of other users than yourself, you must remove this function.

Reply:
Let's also rename setUserStatus to `setMyStatus` and remove the argument. We should get the id like this: `import UsersService from 'app/features/users/services/current-user-service';` and `UsersService.getCurrentUserId();`